ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PullQueryConsistencyFunctionalTest.java

```diff
@@ -344,6 +365,104 @@ public void shouldPassConsistencyTokenInFanOutTableScan()
       ImmutableList.of(KEY4, 1)));
   }
 
+  @Test
```

Comment:
If we allow the `putIfAbsent` in the client, then I can do the tests in this class without doing a raw rest request. More importantly, it will allow me to test the routing to standby functionality as well